src/main/java/org/orienteer/telegram/bot/Cache.java

```diff
@@ -39,7 +39,7 @@ protected Object execute(ODatabaseDocument db) {
     private static void createClassCache(ODatabaseDocument db) {
         classCache = new HashMap<>();
         for (OClass oClass : db.getMetadata().getSchema().getClasses()) {
-            if (OTelegramModule.TELEGRAM_SEARCH.getValue(oClass)) {
+            if (OTelegramCustomAttributes.TELEGRAM_SEARCH.getOrCreate().getValue(oClass)) {
```

Comment:
When we discussed that I thought that you will use CustomAttributes and add "getOrCreate()" method there instead of creating separate class OTelegramCustomAttributes. Any problems with dynamic loading in that case? If no - lets get rid of OTelegramCustomAttributes.